eventmesh-webhook/eventmesh-webhook-receive/src/main/java/org/apache/eventmesh/webhook/receive/storage/WebhookFileListener.java

```diff
@@ -74,9 +75,6 @@ public WebhookFileListener(String filePath, Map<String, WebHookConfig> cacheWebH
      */
     public void filePatternInit() throws FileNotFoundException {
```

Comment:
After updating the code , this FileNotFoundException will never thrown， can be removed

Reply:
OK